com.unity.render-pipelines.high-definition/Runtime/Material/GGXConvolution/IBLFilterGGX.cs

```diff
@@ -209,6 +209,15 @@ void CheckIntermediateTexturesSize(int texWidth, int texHeight)
             }
         }
 
+        void OutputSmoothPlanarReflection(CommandBuffer cmd, RenderTexture sourceColor, RenderTexture target)
+        {
+            int currentTexWidth = sourceColor.width;
+            int currentTexHeight = sourceColor.height;
+
+            // The first color level can be copied straight away in the mip chain. Unfortunately due to a format incompatibility, we have to go through a compute shader copy.
```

Comment:
Is this comment relevant? seems to me we don't use compute here.

Reply:
yup, removed